_docs/create-bitcoin-wallet.md

```diff
@@ -0,0 +1,85 @@
+---
+title: How to create a Bitcoin wallet
+tags: [setup]
+---
+
+Creating a Bitcoin wallet on BlueWallet is easy.
+
+You can either click on the "Add now" button or the "+" icon in the top right corner
+
+{% include phone-screenshot-image.html img="bluewallet-first-page.png" alt="BlueWallet's first page" %}
+
+That will take you to the "add wallet" screen.
+
+## Adding a new wallet
+
+In the "add wallet" screen, you can choose between a Bitcoin wallet and a Lightning wallet.
+
+{% include phone-screenshot-image.html img="add-bitcoin-wallet.png" alt="Add bitcoin wallet page" %}
+
+For Bitcoin, click on the Bitcoin button. (If you want to create a Lightning wallet, see [this guide](/docs/create-lightning-wallet).)
+
+You may also name your wallet, or leave the default name.
+
+<div class="side-note" markdown="1">
+
+### Advanced options
+
+If you have "Advanced mode" enabled in general settings,
+you'll see more options for creating the wallet.
+
+{% include phone-screenshot-image.html img="add-bitcoin-wallet-with-advanced-options.png" alt="Add Bitcoin wallet with advanced options" %}
+
+#### Wallet type
+
+The first group of options is for the wallet type:
+
+##### HD SegWit (BIP84 Bech32 Native)
+
+???
+
+##### SegWit (P2SH)
+
+???
+
+###### HD SegWit (BIP49 PS2H)
+
+???
+
+#### Manual entropy
+
+BlueWallet supports manual entropy. See [here](./manual-entropy.md) for more details.
+
+</div>
+
+When you're ready, you can click on "Create" to make your new wallet.
+
+### Importing a wallet
+
+Instead of creating a new wallet, you can chose to import a wallet from another source or a wallet that you've previously created on BlueWallet.
+
+Click on "Import wallet" to open the import screen.
+
+{% include phone-screenshot-image.html img="import-wallet.png" alt="Import wallet" %}
+
+In this text area you can post your: mnemonic, private key or WIF.
```

Comment:
Can we provide more details for what can be posted here? Mnemonic, private key, or WIF, what else?

Do we always end up with the same type of wallet no matter what the source is? I imagine importing a private key will create a different type of wallet than a mnemonic, no?

Reply:
Also Lightning wallet backup can be imported here (LNDHub authentication)

Imported wallet type depends on a backup. BW will try to guess wallet type.
For example in case of HD backup with 12 words it could be BIP44, BIP49, BIP84 or BRD wallet. BW will try to generate different type of addresses, scan blockchain abd if there are funds on this wallet, it will import it.
If it is BIP44 (legacy addresses started with 1) BW will create wallet of this type.
If you want to migrate funds to newer wallet type, you need to create new wallet and send funds ot it.